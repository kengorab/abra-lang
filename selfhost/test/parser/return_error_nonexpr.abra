return val a = 3