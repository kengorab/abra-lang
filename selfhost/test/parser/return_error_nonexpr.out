Error at %FILE_NAME%:1:8
Unexpected token 'val':
  |  return val a = 3
            ^